package.json

```diff
@@ -3,6 +3,7 @@
   "license": "AGPL-3.0",
   "scripts": {
     "postversion": "git push --tags",
+    "build:babel": "babel app/javascript --out-dir ./build",
```

Comment:
This task for me is mainly for generating the translation strings source (so I can run `yarn manage:translations` with no error). I think it is not necessary for actually generating the webpack bundle.

Open for suggestion how to name / modify it.